.circleci/deploy-to-pantheon.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
> Just talked to an agency that is making the branch, using the multidevs created for them for testing / development, and then creating the PR later when it is ready for review.

This is my experience doing dev work and the common pattern agencies who I interact with are doing as well and what drove this change to begin with.